### PR TITLE
Buffs Brass - Adds a Selection of New Brass Furnishings

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Adds the following structures:

Brass Pews
Pinion Airlock (ported from NSV13, slightly modified)
Brass shelving
Brass Gunrack
Brass Sleeper (not constructible, atomized from an upcoming new ruin PR, ported from NSV13). Produces epinephrine, salbutamol, styptic powder, silver sulfadiazine, oculine, mannitol. Cannot be used without help from a second person. Cannot be moved.

## Why It's Good For The Game
Gives a larger selection of furnishings for enjoyers of clockwork theming, enhancing the depth of builds that can be made. Most importantly, proper airlocks that fit the theme, freeing players from the need to rely on the brass windoor.

## Images of changes
image

## Testing
Grabbed a stack of brass, built every new item.
Deconstructed them afterwards to make sure that worked.
Deconstructed regular shelves and gun racks to make sure THAT still worked.
All was good.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
add: Adds Brass Pews, Pinion Airlocks, Brass Shelves, Brass Gunrack, Brass Sleeper.
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -611,7 +611,15 @@ GLOBAL_LIST_INIT(brass_recipes, list (
 	new /datum/stack_recipe/window("directional brass window", /obj/structure/window/reinforced/clockwork, time = 0 SECONDS, on_floor = TRUE, window_checks = TRUE),
 	new /datum/stack_recipe/window("fulltile brass window", /obj/structure/window/reinforced/clockwork/fulltile, 2, time = 0 SECONDS, on_floor = TRUE, window_checks = TRUE),
 	new /datum/stack_recipe("brass chair", /obj/structure/chair/brass, 1, time = 0 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe_list("pews", list(
+		new /datum/stack_recipe("brass pew (middle)", /obj/structure/chair/sofa/pew/clockwork, 5, one_per_turf = TRUE, on_floor = TRUE),
```
Comment (silverplatedelta) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:615]: Wooden pews are mare out of a single sheet, Five seems to be a bit much

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -373,6 +373,19 @@
 /obj/structure/chair/sofa/pew/right
 	icon_state = "pewend_right"
 
+/obj/structure/chair/sofa/pew/clockwork
+	name = "brass pew"
+	desc = "An ornate pew fashioned from brass. It is even less comfortable than a regular pew, but it does radiate a pleasent warmth."
+	icon_state = "clockwork_pew_middle"
+	buildstacktype = /obj/item/stack/tile/brass
+	buildstackamount = 5
```
Comment (silverplatedelta) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:381]: Same as above here!

**Review by silverplatedelta — COMMENTED** [review]
Other than pews being very expensive, code looks good!

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Fordoxia) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:615]: Wooden pews cost 5 sheets of wood to make, that is what I based the cost on.

Comment (silverplatedelta) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:615]: Seems the wiki is incorrect! It said they were one there. I'll dismiss my comments and update the wiki!

File: code/game/machinery/Sleeper.dm [hunk 3]
```diff
@@ -563,4 +563,29 @@
...
+	component_parts = list()
+	component_parts += new /obj/item/circuitboard/sleeper(null)	// This is going to exist in pre-determined locations only, no special circuit for it.
+	var/obj/item/stock_parts/matter_bin/B = new(null)
+	B.rating = initial_bin_rating
+	component_parts += B
+	component_parts += new /obj/item/stock_parts/manipulator(null)
+	component_parts += new /obj/item/stack/sheet/glass(null)
+	component_parts += new /obj/item/stack/sheet/glass(null)
+	component_parts += new /obj/item/stack/cable_coil(null, 1)
+	RefreshParts()
```
Comment (DGamerL) [inline code/game/machinery/Sleeper.dm:585]:
```suggestion
```

**Review by DGamerL — APPROVED** [review]

**Pull request merged**